src/pages/ProductPage/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,117 @@
+import cn from 'classnames';
+import './Pagination.scss';
+import { useEffect, useState } from 'react';
```

Comment:
Imports should be ordered( you know the order), you are already too precise as a developer, to neglect such 'small' things :)